extensions/gopass/src/create-edit.tsx

```diff
@@ -0,0 +1,226 @@
+import { useEffect, useState } from "react";
+import gopass from "./gopass";
+
+import {
+  Action,
+  ActionPanel,
+  Form,
+  showToast,
+  Toast,
+  Clipboard,
+  closeMainWindow,
+  showHUD,
+  popToRoot,
+} from "@raycast/api";
+import { useForm } from "@raycast/utils";
+
+const createFormText =
+  "Provide name for password and set desired password attributes. Password will be saved to clipboard on creation.";
+const editFormText = "Update desired password attributes. Password will be saved to clipboard on update.";
+
+interface SignUpFormValues {
+  name: string;
+  size: string;
+  length: number;
+  symbols: boolean;
+  digits: boolean;
+  capitalize: boolean;
+  password: string;
+  additionalAttributes: string;
+}
+
+interface renderPasswordProps {
+  type: string;
+  length: number;
+  digits: boolean;
+  symbols: boolean;
+  capitalize: boolean;
+  numbers: boolean;
+}
+async function renderPassword(props: renderPasswordProps): Promise<string> {
+  const passwords = await gopass.pwgen(
+    props.type,
+    props.length,
+    props.symbols,
+    props.digits,
+    props.capitalize,
+    props.numbers
+  );
+  return passwords[Math.floor(Math.random() * passwords.length)].trimEnd();
+}
+
+interface InputProps {
+  inputPassword?: string;
+}
+
+export default function ({ inputPassword = undefined }: InputProps): JSX.Element {
+  const [isLoading, setLoading] = useState<boolean>(true);
+  const [name, setName] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [additionalAttributes, setAdditionalAttributes] = useState<string>("");
+  const [length, setLength] = useState<number>(12);
+  const [digits, setDigits] = useState<boolean>(true);
+  const [symbols, setSymbols] = useState<boolean>(true);
+  const [capitalize, setCapitalize] = useState<boolean>(true);
+  const [numbers, setNumbers] = useState<boolean>(true);
+  const [type, setType] = useState<string>("random");
+  const [showErrors, setShowErrors] = useState<boolean>(true);
+
+  const getInputPassword = async () => {
+    if (!inputPassword) return;
+    const password = await gopass.showAll(inputPassword);
```

Comment:
We could simply use `gopass.password(inputPassword)` and `gopass.show(inputPassword).then((details) => details.join("\n"))`. I'm not sure about this, there's a comment in `gopass.ts` about this :)